.github/workflows/ci.yml

```diff
@@ -10,54 +10,58 @@ jobs:
-
-          - operating-system: 'ubuntu-20.04'
-            php-version: '7.3'
-            job-description: 'with Sf ^5'
-            execute-flex-with-symfony-version: '^5' # Explicit check for Sf 5.x compatibility
-
-          - operating-system: 'ubuntu-20.04'
-            php-version: '8.0'
-            job-description: 'with Sf ^6'
-            execute-flex-with-symfony-version: '^6' # Explicit check for Sf 6.x compatibility
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '7.2'
+#            job-description: 'with lowest deps'
+#            composer-flags: '--prefer-stable --prefer-lowest' # should be checked on lowest supported PHP version
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '7.2'
+#            job-description: 'with Sf ^4'
+#            execute-flex-with-symfony-version: '^4' # Explicit check for Sf 4.x compatibility
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '7.3'
+#            job-description: 'with Sf ^5'
+#            execute-flex-with-symfony-version: '^5' # Explicit check for Sf 5.x compatibility
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '8.0'
+#            job-description: 'with Sf ^6'
+#            execute-flex-with-symfony-version: '^6' # Explicit check for Sf 6.x compatibility
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '7.4'
+#            job-description: 'with deployment'
+#            execute-deployment: 'yes'
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '8.0'
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '8.1'
+#            job-description: 'with migration rules'
+#            execute-migration-rules: 'yes' # should be checked on highest supported PHP version
+#
+#          - operating-system: 'ubuntu-20.04'
+#            php-version: '8.1'
+#            job-description: 'with calculating code coverage'
+#            calculate-code-coverage: 'yes'
+#            phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
 
           - operating-system: 'ubuntu-20.04'
-            php-version: '7.4'
-            job-description: 'with deployment'
-            execute-deployment: 'yes'
-
-          - operating-system: 'ubuntu-20.04'
-            php-version: '8.0'
-            job-description: 'with migration rules'
-            execute-migration-rules: 'yes' # should be checked on highest supported PHP version
-
-          - operating-system: 'ubuntu-20.04'
-            php-version: '8.0'
-            job-description: 'with calculating code coverage'
-            calculate-code-coverage: 'yes'
-            phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
-
-          - operating-system: 'ubuntu-20.04'
-            php-version: '8.1'
+            php-version: 'nightly'
```

Comment:
@SpacePossum , if 8.2 doesn't run properly for us, let's revert, merge this PR without 8.2 in pipeline and work on 8.2 pipeline as separated PR